include/units/Angle.hpp

```diff
@@ -1,25 +1,27 @@
 #pragma once
```

Comment:
Why is units being changed in the PR? Shouldn't it be changed on master then merged into the PR?

Reply:
ideally yes, but units are still considered unstable so I refrained from pushing it to the master branch directly